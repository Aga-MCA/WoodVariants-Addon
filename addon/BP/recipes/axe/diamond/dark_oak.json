{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "diamond_dark_oak_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "minecraft:diamond"
      }
    },
    "result": {
      "item": "aga_wood_variants:diamond_dark_oak_axe"
    }
  }
}