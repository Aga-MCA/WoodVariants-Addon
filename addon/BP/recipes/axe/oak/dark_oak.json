{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "oak_dark_oak_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "aga_wood_variants:oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:oak_dark_oak_axe"
    }
  }
}