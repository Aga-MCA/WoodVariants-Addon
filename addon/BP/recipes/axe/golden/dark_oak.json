{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_dark_oak_axe"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "PP",
      "SP",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:dark_oak_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_dark_oak_axe"
    }
  }
}